api/src/main/java/eu/darkbot/api/managers/ConfigAPI.java

```diff
@@ -169,6 +170,15 @@ default Set<String> getChildren(ConfigSetting<?> root, String path) {
         return null;
     }
 
+    /**
+     * Change current config.
+     * Sample config selector:
+     *   @Option @Dropdown(options = ConfigAPI.class) String conf = "(none)";
+     *
+     * @param config configuration name
+     */
+    void changeConfig(String config);
```

Comment:
```suggestion
    void setConfigProfile(String profile);
```
Should probably start giving these a more proper name of "Config profiles", because right now "config" means way too many things (in-game config 1 or config 2, a config setting, the bot's config tree as a whole, and changing configs as in config profiles)

Reply:
Reverted